app/ui/browser-modern/views/browser/location/security-badge.jsx

```diff
@@ -52,28 +51,24 @@ function mapStateToProps(state, ownProps) {
   const pageLoadState = PagesSelectors.getPageState(state, ownProps.pageId).load;
   const pageErrorState = PagesSelectors.getPageState(state, ownProps.pageId).error;
 
-  // If we have a tofino:// page, if the URL is empty (occurs during
-  // some page loads in our state), or if the page is still loading (many sites
-  // use http -> https redirects, we shouldn't penalize them for a quick interstitial),
-  // just hide the icon
-  let hidden = false;
-  if (!pageLocation ||
-      pageLocation.startsWith(TOFINO_PROTOCOL) ||
+  let image = 'ssl-insecure.svg';
+  if (pageLocation.startsWith(TOFINO_PROTOCOL)) {
+    image = 'logo-tofino.png';
```

Comment:
We have better icons in the branding/iconset directory. Should use one from there.